### ISSUE TITLE
Incorrect MGRA in crossborder model

### ISSUE THREAD

**Issue by bhargavasana** [SANDAG/ABM#100]
Please see maz_id fields in https://github.com/SANDAG/ABM/blob/ABM3_develop/src/asim/configs/crossborder/preprocessing.yaml#L37. I think some of them are incorrect. I believe there are 12 additional MGRAs that correspond to the 12 external TAZs. They are numbered 24322-24333. Please let me know if my understanding is incorrect.
@hannahccarson since you are out, I am assigning this to @aletzdy.

**Comment by aletzdy** [comment]
I need Hannah to confirm (or I will confirm soon myself), but those zones are the internal mapping of the external stations, not the actual external stations. I will confirm soon.

**Comment by hannahccarson** [comment]
Yes, these are the internal zones. Station choice is an accessibility factor to Mexican population which is pre-calculated, so the stations need to be mapped to the internal zones for correct skim properties when choosing internal destination.
